File: Java/Dependency Inversion Principle/Mail.java
public class Mail implements  İletisim {

    public void mailGonder(String mail, String mesaj) {

        System.out.println(mail + " adresine " + mesaj + " mesajı gönderildi");
    }

    @Override
    public void bilgiGonder(Kullanici kullanici, String mesaj) {
        
        mailGonder(kullanici.getMail(), mesaj);
    }
}

File: Python/Dependency Inversion Principle/Mail.py
from İletisim import İletisim
from Kullanici import Kullanici

class Mail(İletisim):
    
    def MailGönder(self,mail, mesaj):
        print(f"{mail} adresine  {mesaj} mesajı gönderildi")
        
    def BilgiGönder(self,kullanici,mesaj):
        self.MailGönder(kullanici.getMail(),mesaj)
